docs/README.md

```diff
@@ -0,0 +1,22 @@
+Simpleflow documentation
+========================
+
+This directory hosts simpleflow documentation.
+
+
+Installing
+----------
+
+    pip install pip-tools
+    ./script/pip-sync
+
+
+Running for development
+-----------------------
+
+    ./script/run
+
+Deploying
+---------
+
+    ./script/deploy
```

Comment:
Yes! That was not finalized but without much feedback I'll add support for readthedocs and it will deploy to both https://simpleflow.readthedocs.io/ (original doc site, common place for python packages docs) and https://botify-labs.github.io/simpleflow/ (nicer place imho)